crates/biome_js_analyze/src/assist/source/organize_imports.rs

```diff
@@ -668,8 +672,9 @@ impl Rule for OrganizeImports {
     fn action(ctx: &RuleContext<Self>, state: &Self::State) -> Option<JsRuleAction> {
         let options = ctx.options();
         let root = ctx.query();
-        let mut organized_items: FxHashMap<u32, JsSyntaxNode> = FxHashMap::default();
-        let mut import_keys: Vec<ImportKey> = Vec::new();
+        let items = root.items().into_syntax();
+        let mut organized_items: FxHashMap<u32, AnyJsModuleItem> = FxHashMap::default();
+        let mut import_keys: Vec<(ImportKey, bool, Option<AnyJsModuleItem>)> = Vec::new();
```

Comment:
It might make sense to use an inline `struct` for this instead of a tuple, so that you have an opportunity to reference the items by name and you can document them. Right now, it's very hard to figure out in the code below what this `bool` even means, and the various `.0`, `.1` and `.2` references don't help with understanding what we're dealing with either.